src/test/java/api/support/APITestContext.java

```diff
@@ -28,7 +28,7 @@
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   private static final String TENANT_ID = "test_tenant";
-  private static final String USER_ID = "79ff2a8b-d9c3-5b39-ad4a-0a84025ab085";
+  public static final String USER_ID = "79ff2a8b-d9c3-5b39-ad4a-0a84025ab085";
```

Comment:
How are the other fields here accessed, is it via properties (get methods)? If so, please introduce one for this rather than public access to the constant.

Reply:
Are there real benefits of getters instead of direct final field access?

Reply:
Only that it hides how these is a static variable. I'm not especially concerned, it was more for consistency (and I don't really mind direct access to immutable state)